packages/grid/test/data-provider.test.js

```diff
@@ -812,14 +812,14 @@ describe('wrapped grid', () => {
 
     it('should clear loading attribute from rows when data received', () => {
       container.dataProvider = (params, callback) => {
-        callback([{}]);
+        callback([{}], 1);
```

Comment:
**note:** The grid size should be updated when the data provider is going to return fewer items than specified by `size`. Otherwise, the virtualizer will always request more items than the data provider can provide.